app/managers.py

```diff
@@ -0,0 +1,38 @@
+from models import Actor
+
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
```

Comment:
The database connection is opened in the constructor. It's important to ensure that this connection is closed when it's no longer needed. Consider adding a method to close the connection, or use a context manager to handle the connection lifecycle.